templates/lune/partials/product/product.snip.html

```diff
@@ -0,0 +1,175 @@
+{{#product}}
+  <main role="main" class="lune-page lune-page-product {{classes}}">
+    <div class="lune-page-product-wrapper md-col-9 mx-auto">
+
+      {{#hero}}
+        <section class="lune-product-section lune-product-section-hero relative z1 flex">
+          <div class="lune-hero-content col-12 md-mt0 flex flex-column items-center justify-center">
+            <div class="lune-hero-title-wrapper">
+              <h1 class="lune-hero-title relative caps m0 px3 md-p0 z1 lune-anim-move-up {{#title_parts.1}}lune-title-has-mobile-hyphen{{/title_parts.1}}" aria-label="{{full_title}}">{{title_parts.0}}</h1>
+              {{#title_parts.1}}
```

Comment:
This title behaviour is unique to the product page hero. I don't think it would be suitable for a partial.